playbooks/generic-build_images.yml

```diff
@@ -11,6 +11,12 @@
         - always
         - imagebuilder
         - upstream_patching
+  tasks:
+    - name: Add utilities repository
```

Comment:
I would prefer if that would be included in the suse-build-image role.

That role is self contained about the setup that needs to happen on a host for building images.

On top of that, that repo should probably in a variable, so that it's easy to override.
That variable can be in the defaults/main.yml of the suse-build-images role.

Reply:
agreed